src/Propel/Generator/Util/QuickBuilder.php

```diff
@@ -606,8 +597,64 @@ public function isIdentifierQuotingEnabled()
      *
      * @return void
      */
-    public function setIdentifierQuoting($identifierQuoting)
+    public function setIdentifierQuoting(bool $identifierQuoting): void
     {
         $this->identifierQuoting = $identifierQuoting;
     }
+
+    /**
+     * Create separate classes to write to physical filesystem.
+     *
+     * @param array $classes
```

Comment:
You can be more concrete here, e.g. string[]
Same for other array usually

Reply:
Fixed.